virttest/utils_misc.py

```diff
@@ -3100,11 +3100,16 @@ def get_image_info(image_file):
                            'dsize' : '931135488',
                            'csize' : '65536'}
     """
+    def check_qemu_image_lock_support():
```

Comment:
Move this implementation to individual method, rather than wrapper in this method

Reply:
@chunfuwen Updated